### ISSUE TITLE
supersid_plot aspect ratio

### ISSUE THREAD

**Issue by kunkleb** [sberl/supersid#9]
The aspect ratio appears to be incorrect when making plots.  I adjusted the values below shown in bold.  This seems to create a better looking plot.  This is roughly below line 230 in supersid_plot.py  

  plot/page size / figure size with on A4 paper
        if len(daysList) == 1:
            fig.set_size_inches(29.7 / 2.54, **15.7** / 2.54, forward=True)
        else:  # allow PDF poster for many days (monthly graph) --> use Adobe PDF Reader --> Print --> Poster mode
            fig.set_size_inches((29.7 / 2.54) * (len(daysList)/2.0), (**15.7** / 2.54) / 2.0, forward=True)
        fig.subplots_adjust(bottom=**0.15**, left=**0.10**, right=0.98, top=0.95)

**Comment by fenrog** [comment]
The size of A4 is 297 mm x 210 mm. [https://www.google.com/search?q=a4+paper+size](url)
The original code uses the correct size of 29.7 cm x 21.0 cm and calculates the size in inches.
Changing 21.0 cm to 15.7 cm would introduce a bug.

```
        # plot/page size / figure size with on A4 paper
        if len(daysList) == 1:
            fig.set_size_inches(29.7 / 2.54, 21.0 / 2.54, forward=True)
        else:  # allow PDF poster for many days (monthly graph) --> use Adobe PDF Reader --> Print --> Poster mode
            fig.set_size_inches((29.7 / 2.54) * (len(daysList)/2.0), (21.0 / 2.54) / 2.0, forward=True)
        fig.subplots_adjust(bottom=0.08, left=0.05, right=0.98, top=0.95)
```
Scaled at 100%, the image with the original size fills exactly one A4 page.
<img width="1044" alt="A4" src="https://user-images.githubusercontent.com/72250570/140055453-3390d768-2b27-4aad-a93f-fc57d4237e85.png">

**Comment by sberl** [comment]
Perhaps we need an option to configure paper size? A4 vs 8.5"x11"

**Comment by fenrog** [comment]
fixed with 95997bd57895d11836e541cc3e1c61c1330a81ce

**Comment by fenrog** [comment]
fixed with 95997bd
